Lib/ufo2ft/__init__.py

```diff
@@ -12,7 +12,7 @@
 
 def compileOTF(ufo, outlineCompilerClass=OutlineOTFCompiler,
                featureCompilerClass=FeatureOTFCompiler, mtiFeaFiles=None,
-               kernWriter=KernFeatureWriter, markWriter=MarkFeatureWriter,
+               kernWriterClass=KernFeatureWriter, markWriterClass=MarkFeatureWriter,
```

Comment:
This is an API change, but having "featureCompilerClass" and "markWriter" as argument when both are classes and not writer instances is confusing and wrong.

Which gets me thinking, if we're going to do some breaking changes, some files and classes could be renamed into something a little more consistent (not that I care).

Either way given that dependencies are pinned changing the API shouldn't be a problem.